### PR TITLE
Updating the readme file of the sample app

### PR DESCRIPTION
**Pull request by eladiw**

### CODE REVIEW

File: sample-datapipeline-app/README.md [hunk 1]
```diff
@@ -1,9 +1,42 @@
...
+Example:
+
+Navigate to the chart's directory and run:
+
+```bash
+helm install .
+--set env.EVENT_HUB_CONSUMER_GROUP="[value]"
+--set env.AZURE_STORAGE_ACCOUNT="[value]"
+--set env.AZURE_STORAGE_ACCESS_KEY="[Base64 value]"
+--set env.AZURE_STORAGE_CONTAINER="[value]" -
```
Comment (devlace) [inline sample-datapipeline-app/README.md]: Minor typo with the "-" in a seperate line

File: sample-datapipeline-app/README.md [hunk 2]
```diff
@@ -1,9 +1,42 @@
...
+- Kubernetes cluster
+
+### Run
+
+Use 'Helm install' command.
+Example:
+
+Navigate to the chart's directory and run:
+
+```bash
```
Comment (devlace) [inline sample-datapipeline-app/README.md:29]: Please add **"\\"** to end of line for single multi-line bash command, like..

helm install . \\
   --set env.EVENT_HUB_CONSUMER_GROUP="[value]" \\
   .... etc...

**Review by devlace — CHANGES_REQUESTED** [review]
Just some minor changes to the helm install command

File: sample-datapipeline-app/README.md (hunk 1 above)
Comment (eladiw) [reply sample-datapipeline-app/README.md]: 👍

File: sample-datapipeline-app/README.md (hunk 2 above)
Comment (eladiw) [reply sample-datapipeline-app/README.md:29]: 👍

**Review by devlace — APPROVED** [review]

**Pull request merged**